plugins/camtrawl/matlab/Processing_with_species_id.m

```diff
@@ -0,0 +1,275 @@
+%   Processing_with_species_id.m
+%   Automated processing of fish in left/right image pairs.
+%   Output is a set of lengths from all the frames and hauls specified.
+%   The lengths are computed by: 
+%               a. Performing a global (full image) threshold subtraction
+%                  based on a running averaged background to create a
+%                  binary image with potential fish objects.  This is done
+%                  for the right and left frame, individually;
+%               b. Each object is labeled, fit with an oriented
+%                  rectangle;
+%               c. The coordinates of the boxes are used to select local
+%                  object frames and threshold according to each individual
+%                  fish's grayscale pixel distribution.  The threshold is
+%                  set to be the mean + 1 standard deviation of the
+%                  gaussian fit to the grayscale values.  Then relabeling
+%                  and re-fitting with an oriented rectange.
+%               d. Fish are matched between the left and right image using
+%                  error threshold from transforming points into 3
+%                  dimensions and then back to 2D pixel space.
+%               e. The length between the mid points of the fit, matched
+%                  boxes are used to find the length of the fish.
+%
+%
+%   Functions required to run:
+%                                extract_targets2.m
+%                                find_match.m
+%                                calssify_species.m
+
+%
+%   3D transformation functions (from camera clibration toolbox - Bouguet 2008):
+%                                stereo_triangulation.m
+%                                project_points2.m 
+%                                   rodrigues.m
+%                                   rigid_motion.m
+%                                   normalize_pixel.m
+%                                       comp_distortion_oulu.m
+%                                       comp_distortion.m
+
+clear all; clc; close all;
+
+%% DEFINE PARAMETERS

```

Comment:
Unexpected line termination. Other places in this file too.

Reply:
Seems to be in the matlab files. At this point these are only here for historical reasons @mattdawkins, can these be deleted?

Reply:
@linus-sherrill The newlines should now be normalized.